core/email_api.php

```diff
@@ -1309,8 +1342,14 @@ function email_bug_reminder( $p_recipients, $p_bug_id, $p_message ) {
 			$t_sender_email = '';
 		}
 		$t_header = "\n" . lang_get( 'on_date' ) . ' ' . $t_date . ', ' . $t_sender . ' ' . $t_sender_email . lang_get( 'sent_you_this_reminder_about' ) . ': ' . "\n\n";
-		$t_contents = $t_header . string_get_bug_view_url_with_fqdn( $p_bug_id ) . " \n\n" . $p_message;
 
+/* Kimberly Keown suggested revision to add a custom email signature. (See Line 522.)
+	Revised Code:*/
+	$t_contents = $t_header . string_get_bug_view_url_with_fqdn( $p_bug_id ) . "\n\n" . $p_ message .  "\n\n" . lang_get( 'email_custom_signature' );
```

Comment:
I'm not sure... does implementing and using the code in my own installation qualify as _testing_??

However, it's true that my bugnote_add_inc.php revision could have been improved to replace the existing class in-line. The email files would require changes to config_defaults_inc.php + language files or custom_strings_inc.php (not included in the current config folder), therefore those additions are referenced only.

As the responses you have posted have been less than constructive, it is clear you are unwilling to fairly review coding that could substantially improve system emails which are sadly lacking in basic customization. That these contributions to an open source program are not welcome is inexplicable. 

My intent in suggesting the revisions is to reciprocate the value I've received from the Bug Tracker, and that remains unchanged. Therefore I am stating my permission to implement the coding I have attempted to provide, at any time now or in the future, without the pull request approval or other acknowledgement.

Thank you for your time.
Kimberly Keown